CMakeLists.txt

```diff
@@ -94,3 +98,23 @@ if(QHOTKEY_INSTALL)
 
     export(TARGETS qhotkey FILE QHotkeyConfig.cmake)
 endif()
+
```

Comment:
Could you please move all next code after the line 24? We prefer having `set` and `include` before.

Reply:
Oh, I wanted to write `include` after `set`. Thanks!